package.json

```diff
@@ -8,7 +8,7 @@
     "performances"
   ],
   "scripts": {
-    "postinstall": "scripts/cli update_submodule",
+    "postinstall": "scripts/cli init_submodule",
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏IIRC, the intent was to be able to detect early if some breaking changes were introduced by the schema.
Maybe we could add a specific gitlab job that run the unit tests on the updated schema.
wdyt?

Reply:
What would be the impact when a breaking change happen in the `rum-events-format`? If we were to catch it when updating the schema in one of our tasks, I think it would be fine, no?

Reply:
The idea would be to detect it sooner than later. If the breaking change is done by the mobile team, it may take us a while before we detect it.

Reply:
We now run SDK tests in `rum-event-format` directly https://github.com/DataDog/rum-events-format/pull/124